### DISCUSSION TITLE
Glossary of techniques to refer to for the preparation steps

### DISCUSSION THREAD

**Discussion by gabt92** [mealie-recipes/mealie#822] [Feature Request]
**Before submitting this feature request I have**
- [X] Reviewed the documentation and verified that my feature request isn't already a feature. 
- [X] Reviewed the existing feature requests and my feature has not been requested. 
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered 

**Please Describe The Problem To Be Solved**
Some recipes can need special techniques and instead of explaining every time in the steps or notes how to process, maybe a dictionary of different techniques and "how to" could be implemented. Quick small example: de-seeding tomatoes.

**(Optional): Suggest A Solution** 
 Create a admin page with different techniques and "how to". All of those techniques could be similar to a recipe in the fact that they could have different steps, pictures or embedded videos. 

In the steps of the recipes needing those techniques, either implement a way to link a technique (like it's done in V1 for the food) or generate something automatic like referring to the title of a technique to automatically create a link to this one.

**Additional Information** 
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature
> I'm welling to help conceptualize and test but I don't think I have the knowledge to help with this one.

**Comment by hay-kot** [comment]
This seems like something I would use notes for, I'd rather have the description in the recipe instead of having to navigate to and from a few different places. 

I'm not convinced this warrants a whole page/wiki like setup.

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened, so I'm not sure if this was an option previously, but you can embed pictures/videos into steps which I think satisfies this request (in addition to notes, as noted above)
